src/mlpack/core/optimizers/sdp/lrsdp_function_impl.hpp

```diff
@@ -81,6 +94,12 @@ void LRSDPFunction<SDPType>::GradientConstraint(
       << "for arbitrary optimizers!" << std::endl;
 }
 
+template <typename SDPType>
+void LRSDPFunction<SDPType>::UpdateRRT(const arma::mat& newrrt) const
+{
+  *(arma::mat*)(&rrt) = newrrt;
+}
+
```

Comment:
This may sound lame but I was unable to update `rrt` through below code:
`rrt = newrrt`
If I am trying to use it then left operand is being retrieved as `const arma::mat` and hence can't change value directly. Please help me out here.